### PR TITLE
Popup modal

### PR DESCRIPTION
**Pull request by ALPANNA-GH**
- create projects object array
- collect project data
- generate recent work section with all project cards by Js code
- create the popup modal window to see more detail of projects
- style popup for both mobile and desktop
- assign corresponding project data to the modal window when clicking `see project`
- close the modal window when click the close button

### CODE REVIEW

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
Hi @ALPANNA-GH :wave: , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

invalid

## Invalid Code Review Request

You have submitted a pull request with no contribution from your coding partner(s). This is a pair-programming activity. Kindly ensure you show your collaboration skills by working as a team with your partner(s) on this project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by ALPANNA-GH** [comment]
@AbayomiOlaoye 
I'm working solo as student success approved according to partner issue of internet
Screenshot 2023-07-28 205028

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @ALPANNA-GH :wave: , 

Good job with this milestone so far! What a brilliant attempt!!
There are just a few more issues to work on before moving on to the next project but you are almost there!

so_close

## Highlights
- Descriptive Pull Request message :heavy_check_mark: 
- Very responsive pop-up :heavy_check_mark:
- Good use of GitHub flow :heavy_check_mark: 
- Meaningful commit messages :heavy_check_mark: 
- No linter error :heavy_check_mark: 
- Descriptive ReadMe documentation :heavy_check_mark: 

## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

- If you don't mind, I think you might have to check your project images because aside from the popup window, no project image is working

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css [hunk 1]
```diff
@@ -384,98 +389,127 @@ input[type='submit']:active {
...
+  display: flex;
+  width: 100%;
+  height: 40%;
+  background-size: 100%;
+  background-position: center;
+  background-repeat: no-repeat;
+  justify-content: center;
+  align-items: center;
+
+  /* overflow: hidden; */
```
[the remark this answers is not in the document]
Comment (ALPANNA-GH) [reply css/style.css:482]: done

File: scripts/script.js [hunk 2]
```diff
@@ -29,6 +29,11 @@ window.onload = function load() {
...
                 standard dummy text ever since the 1500s, 
                 when an unknown printer took a standard dummy text.`,
+      proLongDesc: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+                    Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+                    when an unknown printer took a galley of type and scrambled it to make a type specimen book. 
+                    It has survived not only five centuries, but also the leap into electronic typesetting, 
+                    remaining essent`,
       proImage: '../images/placeholder.svg',
       proTechs: ['CSS', 'HTML', 'Boostrab', 'Ruby'],
       liveLink: 'http://www.github.com/proj1',
```
[the remark this answers is not in the document]
Comment (ALPANNA-GH) [reply scripts/script.js:39]: ok

File: scripts/script.js [hunk 3]
```diff
@@ -163,7 +201,37 @@ window.onload = function load() {
...
+    document.querySelector('#dialogContainer').style.display = 'none';
+    document.querySelector('#bgblur').style.display = 'none';
+  }
+
+  const modalClose = document.querySelector('#dialogCancel');
+  modalClose.addEventListener('click', closePopup);
+  const projectId = document.querySelectorAll('[id^="project"]');
+  projectId.forEach((el) => {
+    el.addEventListener('click', showPopupModal);
+  });
```
[the remark this answers is not in the document]
Comment (ALPANNA-GH) [reply scripts/script.js:232]: done

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @ALPANNA-GH , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Well done implementing the requested changes.  🎉 🎉 🎉 ✔
- I like the animations you've added. 🤗 🤗 🤗 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**